test/parallel/test-debug-v8-fast-api.js

```diff
@@ -0,0 +1,56 @@
+// Flags: --expose-internals --no-warnings --allow-natives-syntax
+'use strict';
+const common = require('../common');
+
+if (!common.isDebug) {
```

Comment:
Would you mind updating v8-fast-api documentation in docs folder as a part of this pull-request as well?